Create TRIAGE_GUIDE.md (#637)
open-sauced/open-sauced: TRIAGE_GUIDE.md

# Open Sauced Triager Guide

## Issue Triage Process

When a new issue or pull request is opened the issue will be labeled with `needs triage`.
If a triage team member is available they can help make sure all the required information
is provided. Depending on the issue or PR there are several next labels they can add for further
classification:

* `needs triage`: This can be kept if the triager is unsure which next steps to take
* `awaiting more info`: If more info has been requested from the author, apply this label.
* `question`: User questions that do not appear to be bugs or enhancements.
* `discuss`: Topics for discussion. Might end in an `enhancement` or `question` label.
* `bug`: Issues that present a reasonable conviction there is a reproducible bug.
* `enhancement`: Issues that are found to be a reasonable candidate feature additions.

> In all cases, issues may be closed by maintainers if they don't receive timely response when

Reviewer: receive **a** timely

> # Express Triager Guide

Reviewer 2: ```suggestion
# Open Sauced Triager Guide
```

## Issue Triage Process

> When a new issue or pull request is opened the issue will be labled with `needs triage`.

Reviewer 2: Typo

```suggestion
When a new issue or pull request is opened the issue will be labeled with `needs triage`.
```

> If a traige team member is available they can help make sure all the required information

Reviewer 2: Typo

```suggestion
If a triage team member is available they can help make sure all the required information
```

is provided. Depending on the issue or PR there are several next labels they can add for further
classification:

* `needs triage`: This can be kept if the triager is unsure which next steps to take
* `awaiting more info`: If more info has been requested from the author, apply this label.

> * `question`: User questions which do not appear to be bugs or enahncements.

Reviewer 2: Typo

```suggestion
* `question`: User questions that do not appear to be bugs or enhancements.
```

* `discuss`: Topics for discussion. Might end in an `enhancement` or `question` label.
* `bug`: Issues that present a reasonable conviction there is a reproducible bug.
* `enhancement`: Issues that are found to be a reasonable candidate feature additions.

In all cases, issues may be closed by maintainers if they don't receive timely response when
further information is sought, or when additional questions are asked.

## Approaches and Best Practices for getting into triage contributions

Review the organization's [ProjectBoard](https://github.com/open-sauced/open-sauced/projects/1),

> pay special attention to the bugs. This gives you an overview of the current bugs and feature in the repository.

Reviewer 2: To Plural

```suggestion
pay special attention to the bugs. This gives you an overview of the current bugs and features in the repository.
```
